lib/graphql/apollo-wrapper.tsx

```diff
@@ -1,8 +1,11 @@
 "use client"
+
+import { createHttpLink } from "@apollo/client";
+import { errorLink, authLink, retryLink } from "@/lib/graphql/graphqlHelper";
```

Comment:
Decided to import some of these links from a new `graphqlHelper` file. That way its easier to test the errorLink.